plaid/processor.go

```diff
@@ -5,21 +5,25 @@ import (
 	"errors"
 )
 
-type createProcessorTokenRequest struct {
+type processorTokenRequest struct {
 	ClientID    string `json:"client_id"`
 	Secret      string `json:"secret"`
 	AccessToken string `json:"access_token"`
 	AccountID   string `json:"account_id"`
+	Processor   string `json:"processor,omitempty"`
 }
 
-type createProcessorTokenResponse struct {
+// ProcessorTokenResponse defines the generic return format for most processor token requests 
+type ProcessorTokenResponse struct {
```

Comment:
consider using `createProcessorTokenRequest` and `CreateProcessorTokenResponse` to match the existing structs, which use the format `<nameOfFunction><Request|Response>`